src/dataset-builder/ConceptSetCreator.ts

```diff
@@ -23,8 +23,18 @@ export const toConceptSet = (concept: Concept): ConceptSet => {
 export const ConceptSetCreator = (props: ConceptSetCreatorProps) => {
   const { onStateChange, dataset, conceptSetUpdater } = props;
   const { snapshotBuilderSettings, id } = dataset;
+  const hierarchyMap = new Map<number, Concept[]>();
+  // create a root for all domainOptions
+  const domainOptionRoot: Concept = {
```

Comment:
I see why this is needed to show the top level domains. And maybe this is the best way to do that! but I'm wondering if you had thought of any other approaches to this problem? Or maybe theres a better place for this root concept to be defined?

Reply:
Yes, we can most likely create a global root? After our discussion with Phil it seems like it's ok to create these 'fake' roots